src/data/mock/systemProfile.ts

```diff
@@ -795,6 +798,32 @@ export const result: { data: GetSystemProfile } = {
         __typename: 'CedarContract'
       }
     ],
+    exchanges: [
```

Comment:
Q: Is it necessary to mock CEDAR on the FE if we already have mocks in the BE?

Reply:
I'm guessing that has to do with the updated associated query and type.

Reply:
I see these mocks are used in UI tests

Reply:
we _could_ rewire the UI tests to ping the BE

Reply:
Ah OK, if it's for FE tests that totally makes sense. Thanks for digging into that!